civis/futures.py

```diff
@@ -221,6 +221,25 @@ def _poll_and_set_api_result(self):
             except Exception as e:
                 self._set_api_exception(exc=e)
 
+    def outputs(self):
+        """
+        Block on job completion and return a list of run outputs if the
+        job is successful.
+
+        Returns
+        -------
+        list[dict]
+            List of run outputs from a successfully completed job.
+
+        Raises
+        ------
+        CivisAPIError
```

Comment:
I think it'd be nice to have this be `civis.base.CivisAPIError` so users don't have to hunt for how to import it.  I don't feel strongly about this since [numpydoc's](https://numpydoc.readthedocs.io/en/latest/format.html#sections) example for "Raises" just says "LinAlgException" rather than the full path.

Reply:
Also, should this be CivisJobFailure instead?  I didn't look into it too deeply, but I'm surprised it'd be CivisAPIError rather than CivisJobFailure. 🤷‍♂

Reply:
Good catch! My text editor autocompleted that one and I didn't notice that it was incorrect.